docs/pages/server-access/guides/tsh.mdx

```diff
@@ -627,10 +627,23 @@ other user can join you through their terminal by typing:
 $ tsh join <session_ID>
 ```
 
-The Teleport user joining the session must have access to the same SSH principal
-used to start the session. In this case, Teleport user `joe` is logged in to the
-SSH Node as Unix user `joe`, so only Teleport users with a role allowing login `joe`
-will be able to join.
+The Teleport user joining the session must have a `join_sessions` policy set up that allows
+them to join sessions created by the other user. If `joe` has a role named `access`, the other
+user needs have a role with a policy as shown below.
+More detailed information about `join_sessions` policies can be found in the [role reference](../../access-controls/reference.mdx) and [moderated sessions guide](../../access-controls/guides/moderated-sessions.mdx).
```

Comment:
I also think we should move this sentence into a separate paragraph below the code snippet, so we go directly from "as shown below" to the snippet.